aiidalab/utils.py

```diff
@@ -172,10 +193,22 @@ def find_installed_packages(python_bin=None):
         encoding="utf-8",
         capture_output=True,
     ).stdout
-    return [
-        Package(name=package["name"], version=package["version"])
-        for package in json.loads(output)
-    ]
+
+    return output
```

Comment:
It makes more sense to me if this method already returned the parsed json object.

```suggestion
    return json.loads(output)
```

Reply:
I second it, changed.